159356-1702-Extramural/capstone
# Setup Selenium test frame work.

Setting up Selenium Python test framework, gives two examples.
How to add new test cases:
1. In test_case dir, create a new py file(or modify an exit one), following the pattern of existing test file.
2. Run the run_all_test.py in dir of seleniumPythonTest.
3. Go to selenium_test_reports dir to see the test result.
